FirebaseAuth/Sources/Utilities/FIRAuthInternalErrors.h

```diff
@@ -482,12 +487,27 @@ typedef NS_ENUM(NSInteger, FIRAuthInternalErrorCode) {
   FIRAuthInternalErrorBlockingCloudFunctionError = FIRAuthPublicErrorCodeFlag |
                                                    FIRAuthErrorCodeBlockingCloudFunctionError,
 
-  /** @var FIRAuthInternalErrorCodeRPCRequestEncodingError
-      @brief Indicates an error encoding the RPC request.
-      @remarks This is typically due to some sort of unexpected input value.
+  /** Indicates that the recaptcha score sent to backend is invalid.
+   */
+  FIRAuthInternalErrorCodeInvalidRecaptchaScore = FIRAuthPublicErrorCodeFlag |
+                                                  FIRAuthErrorCodeCaptchaCheckFailed,
+
+  /** Indicates that the recaptcha integration is not enabled for this project.
+   */
+  FIRAuthInternalErrorCodeRecaptchaNotEnabled = FIRAuthPublicErrorCodeFlag |
+                                                FIRAuthErrorCodeRecaptchaNotEnabled,
 
-          See the @c NSUnderlyingError value in the @c NSError.userInfo dictionary for details.
+  /** Indicates that the recaptcha sdk is not linked to the app.
```

Comment:
Is it SDK linked to the app or app bundle id not linked to the site key?

Reply:
The former. Actually, the latter is a good point. It's handled by recaptcha and we just replay the error message I think.